hax/hax/util.py

```diff
@@ -589,6 +589,62 @@ def get_node_health(self, node: str) -> str:
             raise HAConsistencyException(
                 f'Failed to get {node} node health') from e
 
+    def get_node_fid(self, node: str) -> Optional[str]:
+        # Example,
+        # m0conf/nodes/
+        # 0x6e00000000000001:0x3:{"name": "ssc-vm-1623.colo.seagate.com",
+        #                         "state": "M0_NC_UNKNOWN"}
+        node_items = self.kv.kv_get('m0conf/nodes', recurse=True)
+        for item in node_items:
+            item_value = json.loads(item['Value'])
+            if 'name' in item_value and item_value['name'] == node:
+                return str(item['Key'].split('/')[2])
+        return None
+
+    def get_node_ctrl_fid(self, node: str) -> Optional[str]:
+        # Example,
```

Comment:
Can you add the docstring? I don't quite understand how to provide the proper `node` value. Is it a node hostname?

Reply:
Yes, it's the node hostname. for eg `"ssc-vm-c-1899.colo.seagate.com"` 
`def get_process_node(self, proc_fid: Fid) -> str:` returns the hostname of node from process fid. 

I'll add the docstring 👍

Reply:
done